lib/metrics.js

```diff
@@ -41,65 +42,71 @@ export class Metrics {
       // or values submitted previously
       ...this.subnetMetrics.get(subnetName),
       // or values submitted now
-      ...metrics
+      ...metrics,
     }
 
     if (
       typeof metrics.totalJobsCompleted === 'number' &&
       typeof this.subnetMetrics.get(subnetName)?.totalJobsCompleted === 'number'
     ) {
-      const diff = metrics.totalJobsCompleted -
+      const diff =
+        metrics.totalJobsCompleted -
         this.subnetMetrics.get(subnetName).totalJobsCompleted
       if (diff < 0) {
         this.maybeReportErrorToSentry(
-          new Error(`Negative jobs completed for ${subnetName}`)
+          new Error(`Negative jobs completed for ${subnetName}`),
         )
       } else if (diff > 0) {
         writeClient.writePoint(
           new Point('jobs-completed')
             .tag('subnet', subnetName)
-            .intField('value', diff)
+            .intField('value', diff),
         )
       }
     }
     this.subnetMetrics.set(subnetName, resolvedMetrics)
     const mergedMetrics = {
       totalJobsCompleted: 0,
-      rewardsScheduledForAddress: 0n
+      rewardsScheduledForAddress: 0n,
     }
-    for (const [, metrics] of this.subnetMetrics) {
+    for (const [
+      ,
+      metrics,
+    ] of this.subnetMetrics) {
       mergedMetrics.totalJobsCompleted += metrics.totalJobsCompleted
       // Merging rewards metrics should be revisited as more subnets start
       // paying rewards
       mergedMetrics.rewardsScheduledForAddress +=
         metrics.rewardsScheduledForAddress
     }
-    const isChanged = this.mergedMetrics === null ||
-      Object
-        .entries(this.mergedMetrics)
-        .some(([key, value]) => mergedMetrics[key] !== value)
+    const isChanged =
+      this.mergedMetrics === null ||
+      Object.entries(this.mergedMetrics).some(
+        ([
+          key,
+          value,
+        ]) => mergedMetrics[key] !== value,
```

Comment:
Same here wrt destructuring and multi line arrays

Reply:
Maybe this plugin won't work so well after all, and maybe that's why it's not included in prettier core